sale.xml

```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0"?>
 <tryton>
     <data>
-
```

Comment:
This change was not needed, don't change files unless it is needed by your task.